test/sanity_test.sh

```diff
@@ -6,6 +6,12 @@
 # steps in INSTALL doc.
 #
 # Please feel free to add more basic tests.
+#
+# If user doesn't install the binaries, libraries and head files into the
+# configed path, user must specify path in this way.
+#
+# $sudo LD_LIBRARY_PATH={library path} PATH={bin path}	\
+# C_INCLUDE_PATH={head path} sanity_test.sh
```

Comment:
not sure we need C_INCLUDE_PATH because we need only runtime variables (LD_LIBRARY_PATH and PATH)

Reply:
Yes, we need it since the test file need to include <wd.h>. This "wd.h" is installed from uadk. Without this head file, we'll meet compilation issue.